README.md

```diff
@@ -11,48 +12,53 @@ Create a conda environment in the cloned directory from the `environment.yml` fi
 
     conda env create -f environment.yml
 
-Here, our conda environment name `data_curator_env` is set from the `environment.yml` file . You can change it anything you like, but please note that you will need to make changes accordingly in the `app.R` file.
+Here, our conda environment name `data_curator_env` is set from the `environment.yml` file .
 
 Activate the `data_curator_env` environment:
 
     conda activate data_curator_env
+    
+_Note_:
+- You can change the name of your conda environment inside `environment.yml` or even use another environment, but please note that you will need to make changes accordingly in the `app.R` file line 16.
```

Comment:
..make changes accordingly in the `app.R` file on [line #16](https://github.com/Sage-Bionetworks/HTAN_data_curator/blob/shiny-server-packaged-backend/app.R#L16).

Reply:
I don't often use hyperlinks, would we have to manually update this often? e.g. if we move to shiny server or did other updates?

Reply:
Yes, we would have to update this manually, if ever we decide to import a few more/less libraries and the line that imports the conda environment gets shifted. But I guess that makes sense, it's more manual work for us to update. We don't need to have a hyperlink to the line on GitHub. Feel free to resolve the conversation!

Reply:
if we get to a stable readme on the shiny-server we could add that in or the code example (as long as we don't think it'll change much)